icons/bone.svg

```diff
@@ -9,5 +9,5 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M18.6 9.82c-.52-.21-1.15-.25-1.54.15l-7.07 7.06c-.39.39-.36 1.03-.15 1.54.12.3.16.6.16.93a2.5 2.5 0 0 1-5 0c0-.26-.24-.5-.5-.5a2.5 2.5 0 1 1 .96-4.82c.5.21 1.14.25 1.53-.15l7.07-7.06c.39-.39.36-1.03.15-1.54-.12-.3-.21-.6-.21-.93a2.5 2.5 0 0 1 5 0c.01.26.24.49.5.5a2.5 2.5 0 1 1-.9 4.82Z" />
+  <path d="M9.94 17a1.4 1.4 0 0 0-.1 1.57A2.5 2.5 0 1 1 5 19.5c0-.26-.24-.5-.5-.5a2.5 2.5 0 1 1 1-4.82A1.39 1.39 0 0 0 7 14l7.06-7a1.4 1.4 0 0 0 .14-1.57A2.5 2.5 0 1 1 19 4.5c.01.26.24.49.5.5a2.5 2.5 0 1 1-.9 4.82 1.4 1.4 0 0 0-1.54.18Z" />
```

Comment:
This is the best I could come up with while still keeping the shape exactly 20x20, what do you think?
```suggestion
  <path d="m14.03 6.97-7.06 7.06c-.7.7-1.66-.03-2.47-.03a2.5 2.5 0 0 0 0 5c.28 0 .5.22.5.5a2.5 2.5 0 1 0 5 0c0-.81-.73-1.77-.03-2.47l7.06-7.06c.7-.7 1.66.03 2.47.03a2.5 2.5 0 1 0 0-5 .5.5 0 0 1-.5-.5 2.5 2.5 0 1 0-5 0c0 .81.73 1.77.03 2.47Z" />
```